### PR TITLE
Add the `BaseSell` value for Spool of Wyrdstrand.

### LINKED ISSUE #187: Wyrdstrand sells for 0 to NPC
<a href="https://github.com/Wiggo32"></img></a> **Issue by [Wiggo32](https://github.com/Wiggo32)**
_Thursday Aug 31, 2017 at 06:38 GMT_
_Originally opened as DarkstarProject/darkstar - Issue 4027_

----

**_I have:_**

- [x] searched existing issues (http://github.com/darkstarproject/darkstar - Issue ) to see if the issue I am posting has already been addressed or opened by another contributor
- [x] checked the commit log to see if my issue has been resolved since my server was last updated

**_Client Version_** (type `/ver` in game) **:**30170728_2

**_Server Version_** (type `!revision` in game) **:**unknown

**_Source Branch_** (master/stable) **:**master

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** Spool of Wyrdstrand (item 3550) in the 'item_basic' table has '0' set for BaseSell. Here are 2 sites referencing similar NPC sell prices:
http://ffxiclopedia.wikia.com/wiki/Wyrdstrand
http://www.ffxiah.com/item/3550/wyrdstrand

The Wyrdweave is also set to '0' but I cannot find a reference price online.

### PR DESCRIPTION
**Pull request by mrhappyasthma**
Fixes https://github.com/project-topaz/topaz/issues/187

To apply this change, either run the `sql/item_basic.sql` file, or
just manually execute the following:

```
UPDATE item_basic SET BaseSell = 1319 WHERE name = "spool_of_wyrdstrand"
LIMIT 1
```

**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](https://github.com/project-topaz/topaz/blob/master/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

**Comment by mrhappyasthma** [comment]
I can put the command into a migration script if we felt that was necessary, but might be overkill for this since it's a non-destructive change and can be updated by simply running the existing script again.

**Review by ibm2431 — APPROVED** [review]
Thanks!

No need for a migration script for this sort of thing - those are mainly for schema changes to character related tables where a server operator _can't_ just drop and reimport a table. 😉 

Will hit the merge button when our branch situation is decided and sorted out~

**Comment by ibm2431** [comment]
I ended up holding this at the last meeting to make sure the sale price is based on a fame-less value and we're not accidentally introducing fame into this price. Will merge if the price given in the issue is the right one.

**Comment by 59blargedy** [comment]
I just checked by selling to mog garden (which should be fameless), and the value i have is 1360

**Comment by TeoTwawki** [comment]
> I just checked by selling to mog garden (which should be fameless), and the value i have is 1360

wondering how/where the value of 1319 was captured now, or if mog garden is using an adoulin fame system rendering it non-neutral

**Comment by 59blargedy** [comment]
ok, i made a new character and sent them wyrdstrand. value is 1319. mog garden is NOT fameless.
now to go back and recheck all these item_basic things ive done haha.

**Comment by TeoTwawki** [comment]
_cries in mog garden with his lack of seekers knowledge_

NOOOOOOOOOOoooooooooooooooo a new fame area to add :sob:

**Comment by ibm2431** [comment]
> ok, i made a new character and sent them wyrdstrand. value is 1319

Works for me!

**Pull request merged**
